src/scripts/main.js

```diff
@@ -15,14 +15,26 @@ const food = {
       'Banana': {},
       'Pineapple': {},
     },
-
   },
 };
 
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const ul = document.createElement('ul');
+
+  for (const key in data) {
+    const li = document.createElement('li');
+
+    li.innerText = key;
+    ul.append(li);
+
+    if (Object.keys(data).length !== 0) {
```

Comment:
You should check the length of the keys of the nested object `data[key]` instead of checking the length of keys of the parent object `data`